test/models/vote_test.rb

```diff
@@ -0,0 +1,133 @@
+require "test_helper"
+
+describe Vote do
+
+  before do
+
+    @user_bojack = users(:bojack)
+    @user_diane = users(:diane)
+    @user_todd = users(:todd)
+    @user_jill = users(:jill)
+
+    @work_fully = works(:fully)
+    @work_green = works(:green)
+    @work_breakfast = works(:breakfast)
+    @work_carsick = works(:carsick)
+
+    @vote_1 = Vote.new user_id: @user_jill.id, work_id: @work_carsick.id
+    @vote_2 = Vote.new user_id: @user_todd.id, work_id: @work_breakfast.id
+    @vote_3 = Vote.new user_id: @user_bojack.id, work_id: @work_breakfast.id
+
+  end
+
+  let(:vote) { Vote.new user_id: @user_diane.id, work_id: @work_breakfast.id }
+
+  it "must be valid" do
+    vote = Vote.new user_id: @user_bojack.id, work_id: @work_fully.id
+    value(vote).must_be :valid?
+  end
+
+  it "must have a work_id" do
+
+    vote = Vote.new user_id: @user_bojack.id
+    value(vote).must_be :invalid?
+
+  end
+
+  it "must have a user_id" do
+
+    vote = Vote.new work_id: @work_fully.id
+    value(vote).must_be :invalid?
+
+  end
+
+  it "must have a user_id for an existing user" do
+
+    vote = Vote.new work_id: @work_fully.id, user_id: 27
```

Comment:
**nitpick** Because the testing system assigns a random user id field, it might be technically safer to delete a fixture user and then trying with that old user's id.  

It's __technically__ possible that Minitest could assign a user an id of 27.  Same work work_id below